src/components/ec-file-dropzone/components/ec-upload-cloud-img/ec-upload-cloud-img.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
can we move this file to `src/components/images/ec-upload-cloud.vue`? In case we need to add more in the future. Also I need to be able to determine somehow what component contains plain svgs for a proper webpack bundling in EBO. Having them all together in images folder makes it better.

Reply:
Moved.